### ISSUE TITLE
Add Upload button at /compose route

### ISSUE THREAD

**Issue by tusharkhatriofficial** [tusharkhatriofficial/Blogify#4]
* Add Upload button at `/compose` route.
* Upload the `featured image` to Firebase storage.
* Store the url of uploaded image to MongoDB database. 

Start the convo for more details...

**Comment by Akshay9607** [comment]
i would like to work on this issue , can you please assign this to me

**Comment by rajkumardongre** [comment]
Hi, I would like to work on this issue, can You assign this to me, please?

**Comment by tusharkhatriofficial** [comment]
Yes please... upload the URL to `posts` collection with the title of `postFeaturedImage` and write the steps to setup firebase storage in `Installation` section of the README.md file.

Acc. to me the best way to approach this problem is:

* Once the admin clicks the `Publish` button the image url entered in the text-box must be stored in firebase storage and the URl be stored in the MongoDb database.
* In the next step, add the button to upload the image... The url must be stored only when the admin clicks on the `Publish` button.
